nemo/collections/nlp/data/token_classification/punctuation_capitalization_tarred_dataset.py

```diff
@@ -897,8 +897,14 @@ def __init__(
         world_size: int = 1,
         global_rank: int = 0,
         shuffle_n: int = 1,
+        shard_strategy: str = "scatter",
     ) -> None:
         super().__init__()
+
+        valid_shard_strategies = ['scatter', 'replicate']
+        if shard_strategy not in valid_shard_strategies:
+            raise ValueError(f"`shard_strategy` must be one of {valid_shard_strategies}")
```

Comment:
```suggestion
            raise ValueError(
                f"Invalid shard strategy of type {type(shard_strategy)} "
                f"{repr(shard_strategy) if len(repr(shard_strategy)) < 100 else repr(shard_strategy)[:100] + '...'}! "
                f"Allowed values are: {valid_shard_strategies}."
            )
```

Reply:
Ok. I've propagated the change to other tarred datasets, so as to keep the error the same on all occasions.